infra/base-images/base-runner/bad_build_check

```diff
@@ -165,13 +165,32 @@ function check_mixed_sanitizers {
   fi
 }
 
+# Verify that the given fuzz target doesn't crash on the seed corpus.
+function check_seed_corpus {
+  local FUZZER=$1
+  local FUZZER_OUTPUT="/tmp/$(basename $FUZZER).output"
+
+  # Set up common fuzzing arguments, otherwise "run_fuzer" errors out.
+  if [ -z "$FUZZER_ARGS" ]; then
+    export FUZZER_ARGS="-rss_limit_mb=2048 -timeout=25"
+  fi
+
+  bash -c "run_fuzzer $(basename $FUZZER) -runs=0" &> $FUZZER_OUTPUT
```

Comment:
-runs=0 wont work for afl, hongfuzz (also i think those rss_limit_mb, timeout one), so code might just get stuck there infinitely. i think you need the same fragment that test_all had before. then everything works.

-    if [[ "$FUZZING_ENGINE" = honggfuzz ]]; then	
-      timeout --preserve-status -s INT 20s run_fuzzer $FUZZER	
-    elif [[ "$FUZZING_ENGINE" = libfuzzer ]]; then	
-      run_fuzzer $FUZZER -max_total_time=20	
-    else	
-      export AFL_NO_UI=1	
-      timeout --preserve-status -s INT 20s run_fuzzer $FUZZER	
-    fi

Reply:
Good point. Actually, `test_all` calls `bad_build_check` only for libFuzzer builds, but I've just added a sanity check here as well to prevent any problems in case someone tries to perform `bad_build_check` for a single non-libFuzzer target.

As for AFL and honggfuzz cases, I'm not sure whether those command still work or not, and would rather not mess with them right now. Hopefully @jonathanmetzman and @robertswiecki will add support for AFL and honggfuzz in future.